content/menu.md

```diff
@@ -4,15 +4,19 @@ site_menu:
     url: /get-started
     color: orange-70
     icon: rocket_launch
-  - category: Get started / Installation
+  - category: Get started / Setup
     url: /get-started/installation
-  - category: Get started / Installation / Use command line
+  - category: Get started / Setup / LOCAL DEVELOPMENT
+  - category: Get started / Setup / Use Streamlit Playground ✨
```

Comment:
Shouldn't this be under `CLOUD DEVELOPMENT` ?

Reply:
Still feels funny...

I think the first under the "set up" section needs to be how to install locally. Also, I guess the real problem is that it doesn't really feel like the playground belongs in the Setup section. 

What if we pulled it out of that menu and renamed the menu to Installation?

* Get started
  * Playground
  * Installation
    * LOCAL DEVELOPMENT
    * Install via command line
    * Install via Anaconda Distribution
    * CLOUD DEVELOPMENT
    * Use GitHub Codespaces
    * Use Snowflake
  * Fundamentals
  * First steps
* Develop
* Deploy
* Knowledge base